tests/pep8

```diff
@@ -0,0 +1,15 @@
+# This file should be used as the pep8 and autopep8 config
+# The default config location for these two tools is ~/.config/pep8
+[pep8]
+# Pep8 codes:
+# E111 - indentation is not a multiple of four
+# E114	indentation is not a multiple of four (comment)
+#
+# autopep8 codes:
+# E101 - reindent all lines: removed because max_line_length
+# E501 - Try to make lines fit within --max-line-length characters: removed because max_line_length
```

Comment:
Why do you remove this? it will take into account --max_line_length and make continuation lines... Not always very aesthetic though...

Reply:
Because I thought it was decided to have line length of 120 ? If not, we can just add it, I am fine with it

Reply:
Yes, this is what I was saying: if you set max_line_length to 120, it will try and make them shorted by breaking the lines... However the place of the break is not necessarily where aesthetically you would put it yourself...

Reply:
OK so, do we disable it, or do we let autopep8 do it ? I have no strong opinion there. I'd say that at least, if we let autopep8 do the job, we'll all have the same

Reply:
let autopep8 do the job

Reply:
Sorry, I may have been mislead by what you meant here @chaen, as these lines are comments, and I had understood that this was preventing autopep8 to do the job... I agree autopep8 does the job... One can still modify it by hand if the continuation location doesn't seem appropriate

Reply:
I'll modify the comments to make them clearer